src/tiles/clickable-tile.component.ts

```diff
@@ -33,6 +35,11 @@ import { Router } from "@angular/router";
 	</a>`
 })
 export class ClickableTile {
+	@HostBinding("class.bx--tile--light") get lightThemeEnabled() {
+		return this.theme === "light";
+	}
```

Comment:
Do we need to assign the class to the host or the anchor link? Class is being assigned to the host & anchor link (see line 28).

Pretty sure you do not need to assign to the host.

Reply:
Also if you don't need hostbinding after, please remove host import in line 7